src/materialized/tests/pgwire.rs

```diff
@@ -244,6 +244,16 @@ fn test_multiple_statements() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+#[test]
+fn test_simple_query_no_hang() -> Result<(), Box<dyn Error>> {
+    let (_server, mut client) = util::start_server(util::Config::default())?;
```

Comment:
In theory should should start with `ore::test::init_logging();`, but it's not a big deal.